### DISCUSSION TITLE
is it possible to stream video and audio from diffrent source?

### DISCUSSION THREAD

**Discussion by Monirzadeh** [anilbeesetti/nextplayer#774] [Q&A]
Hi.
is it technically possible to nextplayer can stream Video and audio from different source and use that audio for specific video.
why it can be helpful?
if you stream some service like youtube with external player some resolution don't have audio tracks. this help user to stream that resolution with audio track.

**Comment by anilbeesetti** [comment]
@Monirzadeh  currently not supported. But i'll look into it after all main functionality of the video player is completed
